### ISSUE TITLE
Font has really bad artifacts when running scaled up

### ISSUE THREAD

**Issue by JoshuaWebb** [JoshuaWebb/nullpomino#95]
1. Options > General Options > Screen Size: 1024x768
2. Restart

**Expected:**
Nice looking fonts

**Actual:**
Grubby looking fonts... I mean, look at this jank:
![artifacts-1](https://cloud.githubusercontent.com/assets/6722399/14408583/da77b56a-ff3d-11e5-991b-774a1678f602.PNG)
![artifacts-2](https://cloud.githubusercontent.com/assets/6722399/14408584/daa106d6-ff3d-11e5-94fc-608e94eaf822.PNG)

**Comment by PiotrGrochowski** [comment]
I think the font is stored in an image. So for example Y is next to Z. Because bilinear filter blurs the image, left part of Z is visible in Y.

**Comment by JoshuaWebb** [comment]
Yeah, I've had this fixed in some capacity on my local for about a year but I've never published it.

The same goes for the blockskin, using nearest neighbor fixes the issue at particular resolutions, but doesn't work as nicely for the half-step resolutions. Padding the skins with a 1 pixel radius and keeping the `Image.FILTER_LINEAR` looks nicer for most skins at these intermediate resolutions, but obviously isn't as crispy.

A more complete solution may be a toggle between the two strategies based on the current resolution, with an option to explicitly override based on personal preference.

That said, I don't intend on being the source of truth for ongoing nullpomino maintenance, that's a pretty big responsibility and I just don't have the time or knowledge to commit to that at this point in time.

**Comment by PiotrGrochowski** [comment]
Maybe would you also like to see [my issues](https://github.com/JoshuaWebb/nullpomino/issues/98)?
